### PR TITLE
Event counter listener

### PR DESCRIPTION
**Pull request by twenzel**
Fix comments from PR

### CODE REVIEW

File: src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/EventCounterListener.cs [hunk 1]
```diff
@@ -195,87 +209,29 @@ private void ExtractAndPostMetric(string eventSourceName, IDictionary<string, ob
                             EventCountersInstrumentationEventSource.Log.EventCounterRefreshIntervalLessThanConfigured(actualInterval, this.options.RefreshIntervalSecs);
                         }
                     }
-                    else if (key.Equals("Count", StringComparison.OrdinalIgnoreCase))
```
Comment (hananiel) [inline src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/EventCounterListener.cs:198]: Why did you remove this?

File: src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/MeterTelemetryPublisher.cs [hunk 2]
```diff
@@ -57,19 +56,16 @@ private void StoreValue(MetricKey metricKey, MetricInstrument instrument, Metric
...
-                    this.longValueStore[metricKey] = (long)metricTelemetry.Sum;
+                case InstrumentationType.Counter:
+                case InstrumentationType.Gauge:
+                    this.longValueStore[metricKey] = (long)metricTelemetry.Value;
                     break;
-                case MetricType.DoubleGauge:
-                case MetricType.DoubleSum:
-                    this.doubleValueStore[metricKey] = metricTelemetry.Sum;
-                    break;
-                case MetricType.Histogram:
```
Comment (hananiel) [inline src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/MeterTelemetryPublisher.cs:68]: How did you make sure this is not needed?

**Review by hananiel — COMMENTED** [review]
I have a couple of questions in there but looks good.

File: src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/EventCounterListener.cs (hunk 1 above)
Comment (twenzel) [reply src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/EventCounterListener.cs:198]: It was not used. So why reading it from the payload.

File: src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/MeterTelemetryPublisher.cs (hunk 2 above)
Comment (twenzel) [reply src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/MeterTelemetryPublisher.cs:68]: cijo mentioned that histogram is not needed

File: src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/EventCounterListener.cs (hunk 1 above)
Comment (hananiel) [reply src/OpenTelemetry.Contrib.Instrumentation.EventCounters/Implementation/EventCounterListener.cs:198]: Oh I get it now -thanks.
